test/test.py

```diff
@@ -0,0 +1,168 @@
+import logging
+import mock
+import yaml
+import test_mock
+import unittest
+from unittest import TestCase
+from reviewrot.githubstack import GithubService
+from reviewrot.gitlabstack import GitlabService
+from reviewrot.pagurestack import PagureService
+from github.GithubException import BadCredentialsException
+from gitlab.exceptions import GitlabConnectionError
+
+# Disable logging to avoid messing up test output
+logging.disable(logging.CRITICAL)
+
+
+class GithubTest(TestCase):
+    def test_object_create(self):
+        isinstance_ = isinstance(GithubService(), GithubService)
+        self.assertEqual(isinstance_, True)
+
+    def test_request_review_token(self):
+        with open('test_githubtest.yaml', 'r') as f:
```

Comment:
I would recommend a small stylistic change here.

You can move this config loading into a `setUp(self)` method on the class so that you only have to write it once.

Assign config to ``self.config`` in the setUp method, and then reference that in the tests.  The setUp method will be called before every test.

(There is also a ``tearDown(self)`` method that is run *after* every test that you can use to clean things up if you need to.  You don't need to in this situation.)